src/groupByKey.ts

```diff
@@ -2,6 +2,18 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey<T>(items: T[], key: keyof T): GroupsMap<T> {
+  const groupedObjects: GroupsMap<T> = {};
+
+  items.forEach((item) => {
+    const value: string = String(item[key]);
```

Comment:
```suggestion
    const value = String(item[key]);
```
примитивы указывать не нужно, они наследуются из значения